lib/github/templates/security-post-release.md

```diff
@@ -0,0 +1,18 @@
+---
+date: %ANNOUNCEMENT_DATE%
+category: vulnerability
+title:  %RELEASE_DATE% Security Releases
+slug: %SLUG%
+layout: blog-post.hbs
+author: %AUTHOR%
```

Comment:
The Node.js Project

Reply:
I checkedbthe past releases we put the author there.